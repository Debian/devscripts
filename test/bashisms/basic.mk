#!/usr/bin/make -f

# bug:
overrideSHELL := bash

test:
	-echo -e "foo BASHISM"
	@echo -e "bar BASHISM"
	@-echo -e "bar BASHISM" && false
	-@echo -e "bar BASHISM" && false
	true

dirs:
source diff:
source diff.gz::
source file-stamp:
caller %.so:
	:

foo: $(shell echo dir/BASHISM/{foo,bar})
	:
