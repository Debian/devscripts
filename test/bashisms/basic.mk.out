possible bashism in bashisms/basic.mk line 7 (echo -e):
	-echo -e "foo BASHISM"
possible bashism in bashisms/basic.mk line 8 (echo -e):
	@echo -e "bar BASHISM"
possible bashism in bashisms/basic.mk line 9 (echo -e):
	@-echo -e "bar BASHISM" && false
possible bashism in bashisms/basic.mk line 10 (echo -e):
	-@echo -e "bar BASHISM" && false
possible bashism in bashisms/basic.mk line 20 (brace expansion):
foo: $(shell echo dir/BASHISM/{foo,bar})
